lib/decidim/duplicates_metadata.rb

```diff
@@ -0,0 +1,62 @@
+# frozen_string_literal: true
+
+module Decidim
+  class DuplicatesMetadata
+    def initialize
+      @authorizations = Decidim::Authorization.where(name: %w(data_authorization_handler extended_socio_demographic_authorization_handler))
+    end
+
+    def perform
+      update_authorized_users
+    end
+
+    def update_authorized_users
```

Comment:
`updated_authorized_users` and `update_metadata` are not easy to read, we should consider refactor to keep it simple